rcl/src/rcl/logging.c

```diff
@@ -43,6 +44,8 @@ static bool g_rcl_logging_stdout_enabled = false;
 static bool g_rcl_logging_rosout_enabled = false;
 static bool g_rcl_logging_ext_lib_enabled = false;
 
+static atomic_uint_least64_t __rcl_logging_init_count = ATOMIC_VAR_INIT(0);
```

Comment:
Don't use `__`, that's reserved by the C/C++ standard. Follow the style in this file and use `g_`.

Reply:
got it, will be fixed into the next commit.